package.json

```diff
@@ -9,9 +9,10 @@
   ],
   "homepage": "https://vega.github.io/vega-lite/",
   "description": "Vega-Lite is a concise high-level language for interactive visualization.",
-  "main": "build/src/index.js",
+  "main": "build/vega-lite.js",
```

Comment:
This may break a lot of things?

Reply:
This is consistent with Vega, though. https://github.com/vega/vega/blob/master/package.json